lms/djangoapps/bulk_email/tests/test_email.py

```diff
@@ -297,6 +297,25 @@ def test_send_to_staff(self):
             [self.instructor.email] + [s.email for s in self.staff]
         )
 
+    @patch('bulk_email.tasks.is_email_use_default_from_bulk_enabled', Mock(return_value=True))
```

Comment:
Can I suggest overriding the setting instead of mocking the `is_email_use_default_from_bulk_enabled` function?